packages/shared/src/types/notification-templates/index.ts

```diff
@@ -33,3 +33,8 @@ export enum WorkflowTypeEnum {
   ECHO = 'ECHO',
   BRIDGE = 'BRIDGE',
 }
+
+export enum WorkflowOriginEnum {
+  NOVU = 'novu',
+  EXTERNAL = 'external',
+}
```

Comment:
```suggestion
/**
 * Enum to define the origin of the workflow.
 * 
 * The `WorkflowOriginEnum` is used to specify the source of the workflow,
 * which helps determine which endpoint to call during the Preview & Execution phase.
 * - 'novu' indicates that the workflow originates from Novu's platform, so the Novu endpoint is used.
 * - 'external' indicates that the workflow originates from an external source, requiring a call to a customer-hosted Bridge endpoint.
 */
export enum WorkflowOriginEnum {
  NOVU = 'novu',
  EXTERNAL = 'external',
}

export enum WorkflowOriginEnum {
  NOVU = 'novu',
  EXTERNAL = 'external',
}
```

Reply:
add details on the origin in the entity and DTO based on the following explanation